### DISCUSSION TITLE
Add option to `winter:version` command to get only the build number

### DISCUSSION THREAD

**Discussion by damsfx** [wintercms/winter#1195] [Ideas]
In a bash process, I need to get the current Winter version, but the `winter:version` command is returning a full sentence: 
> *** Detected Winter CMS build 1.2.6.

Actually I did it by getting the last 6 chars of the command output.

What do you think of adding an new option to this command to only get the current build number?

**Comment by mjauvin** [comment]
You should be able to get it from `System\Classes\UpdateManager`, no ?

**Reply by damsfx** [reply]
Sure, but can't use it in a bash script on the server (kind of Winter utilities CLI).
![image](https://github.com/user-attachments/assets/30bb933e-c6bb-4a1f-868d-92281aadc7ba)
![image](https://github.com/user-attachments/assets/3e413c6d-80bd-4a96-9d70-001dd82a27d8)


I was wondering about that before I went ahead and created a plugin just for that.

**Reply by LukeTowers** [reply]
@damsfx do you want to add a PR to the Winter CLI for your utilities?

**Reply by damsfx** [reply]
@LukeTowers My utilities are mainly shortcuts for several artisan commands, allowing me to simplify entering long commands.
Eg: `purgeAll` => `php artisan winter:util purge thumbs --force && php artisan winter:util purge uploads --force && php artisan winter:util purge orphans --force`

You think this could be useful to other users?

**Reply by bennothommo** [reply]
@damsfx I think we would rather have too many features rather than too few :stuck_out_tongue_winking_eye:

**Comment by mjauvin** [comment]
@damsfx care to submit a PR to winter:version to add an option to only return a robot-style version ?

**Comment by bennothommo** [comment]
I've added a flag to do this in https://github.com/wintercms/winter/commit/6d59b82c21442835b9e78b0ac97f7dc58497e84f. Should be available with the next release.
